dbms/src/Flash/Coprocessor/DAGExpressionAnalyzer.cpp

```diff
@@ -332,11 +334,15 @@ void DAGExpressionAnalyzer::buildCommonAggFunc(
     Names arg_names;
     DataTypes arg_types;
     TiDB::TiDBCollators arg_collators;
-    for (Int32 i = 0; i < child_size; ++i)
+
+    // if count(not null const), just remove the argument.
```

Comment:
Please add unit tests in https://github.com/pingcap/tiflash/blob/478f3050cf03151512985fb53343e1c47219913b/dbms/src/Flash/tests/gtest_aggregation_executor.cpp.
Like `count(not_null_const )`, `count(null)`

Reply:
More comments since the modify is related to TiDB